features/step_definitions/work_steps.rb

```diff
@@ -249,6 +249,29 @@
   click_button("Preview")
 end
 
+When /^I set the fandom to "([^\"]*)"$/ do |fandom|
+  fill_in("Fandoms", :with => fandom)
+end
+
+When /^I select "([^\"]*)" for editing$/ do |title|
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.